### PR TITLE
Bug fix: Opening search pulls viewport to top of screen.

### PR DESCRIPTION
**Pull request by jamenlyndon**

### CODE REVIEW

File: src/components/helpers/Overlay/style.scss [hunk 1]
```diff
@@ -30,8 +30,5 @@
 
 .Overlay-open {
   overflow: hidden;
-  position: fixed;
-  top: 0;
-  bottom: 0;
```
Comment (davidherse) [inline src/components/helpers/Overlay/style.scss]: Will removing these work? How does the overlay position itself in front of the content now?

**Comment by benoneal** [comment]
Wait, what's the bug? That the screen scrolls to the top when the lightbox opens?

**Comment by jamenlyndon** [comment]
Yep. When you open search, the screen scrolls to the top. Removing this CSS fixes the issue - not exactly sure why it was required in the first place?

**Comment by davidherse** [comment]
Where does this scroll to top behaviour exist? I can't see it on the example page.

**Comment by jamenlyndon** [comment]
Here's an example:
http://everydayhero.github.io/public-api-docs/#pagesearch

File: src/components/helpers/Overlay/style.scss (hunk 1 above)
Comment (jamenlyndon) [inline src/components/helpers/Overlay/style.scss]: This class is only applied to the body. The body doesn't need to be position fixed, that's crazy stuff.

**Comment by jamenlyndon** [comment]
screenshot

A fix for a scroll bug on windows has also been added to this branch. It removes the unnecessary scrollbar in all browsers and in IE8, allows you to scroll the results (without this fix you can't scroll at all).

Changed "overflow-y" to "auto" for the following classes:
.SearchResults
.SearchModal__body

**Pull request closed without merging**